### PR TITLE
Bugfixes with blacklist

### PR DESCRIPTION
**Pull request by Nyholm**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no (has not been released yet)
| Deprecations?   | no
| Related tickets | 
| Documentation   | 
| License         | MIT

This will fix #62 

The BC breaking change is that we allow any regex delimiter.

### CODE REVIEW

File: src/CachePlugin.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ final class CachePlugin implements Plugin
      *              we have to store the cache for a longer time than the server originally says it is valid for.
      *              We store a cache item for $cache_lifetime + max age of the response.
      *     @var array $methods list of request methods which can be cached
-     *     @var array $blacklisted_paths list of regex patterns of paths explicitly not to be cached
+     *     @var array $blacklisted_paths list of regex of to URLs explicitly not to be cached
```
Comment (gmponos) [inline src/CachePlugin.php]: Shouldn't it be just `of URLs`

**Review by gmponos — APPROVED** [review]
A small typo change only.

Although it is not related with the current PR please consider at some point refactoring the tests and using actual objects.. For instance if you used `ArrayAdapter` and an actual `Request` object I believe some of the tests changes there would not be needed now.

Anyway as I said it is not blocking for the current PR.

File: src/CachePlugin.php (hunk 1 above)
Comment (Nyholm) [reply src/CachePlugin.php]: Oh,, I'll go with "for"

**Comment by Nyholm** [comment]
Yeah, But that is not how PHPSpec is working (as far as I know). But I would love to do that. =)

Thank you for the review

**Pull request merged**